source/includes/steps-export-filter.yaml

```diff
@@ -1,46 +1,70 @@
-title: Navigate to the :doc:`collection </collections>` you wish to export data from.
+title: Navigate to the :doc:`collection </collections>` you wish to 
+       export data from.
 level: 4
 ref: select-collection
 content: |
   You can either select the collection from the
   :doc:`Collections </collections>` tab or click the collection in the
   left-hand pane.
+
 ---
 title: "Specify a filter in the Query Bar"
 level: 4
 ref: export-specify-filter
 content: |
-  Specify a :ref:`filter <query-bar-filter>` in the :ref:`query bar <compass-query-bar>` to export only the documents which match
-  the filter.
+  Specify a :ref:`filter <query-bar-filter>` in the 
+  :ref:`query bar <compass-query-bar>` to export only the documents 
+  which match the filter.
+
 ---
-title: Click :guilabel:`Collection` in the top-level menu and select :guilabel:`Export Collection`
+title: Click :guilabel:`Collection` in the top-level menu and select 
+       :guilabel:`Export Collection`
 level: 4
 ref: export-collection
 content: |
-
-  |
-
   .. figure:: /images/compass/export-data-filter-option-select.png
      :alt: Export data select menu
 
   |compass-short| displays the following dialog:
 
   .. figure:: /images/compass/export-data-dialog-filter.png
      :alt: Export data dialog
-     :figwidth: 600px
+     :width: 60%
 
   The top section of the export dialog displays the query
-  entered in the query bar.
+  entered in the query bar. If you did not specify a query in the query 
+  bar, you can specify a query here.
+
 ---
-title: Choose the appropriate file type.
+title: Select document fields to include in your exported file.
 level: 4
-ref: click-file-type
+ref: select-export-fields
 content: |
-    Under :guilabel:`Select Output File Type`, select either JSON or CSV.
+  Only fields that are checked are included in the exported file.
+
+  You can add document fields to include with the :guilabel:`Add Field` 
+  button if the field you want to include is not automatically detected.
```

Comment:
You might want to explain the sampling behavior a bit here. Users might wonder why their field wasn't detected.

Reply:
I don't think this has to do with sampling because you're exporting a full collection or a partial collection defined by a filter. I think Compass may leave out fields that don't appear on some % of documents. I'll call this out in tech review as well and add some context.

Reply:
My mistake, it does detect fields by sampling. Will add some context like you suggested.